algorithms/CPlusPlus/Strings/print-alphabets.cpp

```diff
@@ -0,0 +1,33 @@
+/*
+Description: A C++ program to print first letter of every word in a string
+*/
+
+
+//importing c++ libraries
+#include <bits/stdc++.h>
+#include <iostream>
+using namespace std;
+
+//function
+string firstAlphabet(string s){
+  //declaring an output string 'o'
+string o;
+//inserting 1st letter in the output string
+	  o+=s[0];
+    //iterating over the string from index 1
+    //if space is found, then the next character is appended in the output string
+	  for(int i=1;i<s.size();i++){
+	      if(s[i]==' '){
```

Comment:
what if there are more than one space between 2 words ?

Reply:
updated the code for that too